### ISSUE TITLE
Support `\prl` `\pra` and the `ruledtabular` environmen from revtex 

### ISSUE THREAD

**Issue by cgogolin** [cgogolin/quantum-journal#54]
quantum article should support there commands and maybe throw a warning (in case of `ruledtabular`) if they are used.

**Comment by cgogolin** [comment]
We need to have a look into the revtex docs what they do.

**Comment by johannesjmeyer** [comment]
This is mostly a compatibility thing, isn't it? And what do you mean with `\prl` and `\pra`? I found no such commands in the revtex docs.

Here is what the docs say about `\ruledtabular`:

> The heart of the table is the tabular environment.
This will behave for the most part as in standard LATEX 2ε
(please refer to Section 3.6.3 and Appendix C.10.2
of the LATEX User’s Guide & Reference Manual for
more details about the tabular environment). Note
that REVTEX 4.1 no longer automatically adds double
(Scotch) rules around tables. Nor does the tabular environment
set various table parameters for column spacing
as before. Instead, a new environment ruledtabular
provides this functionality. This environment should surround
the tabular environment[:]

This means we should provide an environment that gives you lines around your tables.

**Comment by cgogolin** [comment]
The `\pra` `\prl` and similar commands indeed seem to not be documented! They are defined in `revtex4-1/source/latex/revtex/revtex4-1.dtx` in a section that looks like this: 
```
% \subsection{Abbreviations}%
% \changes{4.1p}{2010/02/24}{(AO, 584) Per MD, remove trailing space character from each journal abbreviation: it had caused an extraneous space in the .bbl} 
% The following macros constitute typing shortcuts for
% certain journal names.
%    \begin{macrocode}
\def\ao{Appl.\  Opt.}%
\def\ap{Appl.\  Phys.}%
\def\apl{Appl.\ Phys.\ Lett.}%
\def\apj{Astrophys.\ J.}%
\def\bell{Bell Syst.\ Tech.\ J.}%
\def\jqe{IEEE J.\ Quantum Electron.}%
\def\assp{IEEE Trans.\ Acoust.\ Speech Signal Process.}%
\def\aprop{IEEE Trans.\ Antennas Propag.}%
\def\mtt{IEEE Trans.\ Microwave Theory Tech.}%
\def\iovs{Invest.\ Ophthalmol.\ Vis.\ Sci.}%
\def\jcp{J.\ Chem.\ Phys.}%
\def\jmo{J.\ Mod.\ Opt.}%
\def\josa{J.\ Opt.\ Soc.\ Am.}%
\def\josaa{J.\ Opt.\ Soc.\ Am.\ A}%
\def\josab{J.\ Opt.\ Soc.\ Am.\ B}%
\def\jpp{J.\ Phys.\ (Paris)}%
\def\nat{Nature (London)}%
\def\oc{Opt.\ Commun.}%
\def\ol{Opt.\ Lett.}%
\def\pl{Phys.\ Lett.}%
\def\pra{Phys.\ Rev.\ A}%
\def\prb{Phys.\ Rev.\ B}%
\def\prc{Phys.\ Rev.\ C}%
\def\prd{Phys.\ Rev.\ D}%
\def\pre{Phys.\ Rev.\ E}%
\def\prl{Phys.\ Rev.\ Lett.}%
\def\rmp{Rev.\ Mod.\ Phys.}%
\def\pspie{Proc.\ Soc.\ Photo-Opt.\ Instrum.\ Eng.}%
\def\sjqe{Sov.\ J.\ Quantum Electron.}%
\def\vr{Vision Res.}%
%    \end{macrocode}
```
I suggest we simply copy all the non-commented definitions into a bloc that is executed only if we are in revtex compatibility mode. Does that sound good?

**Comment by johannesjmeyer** [comment]
Sounds good! I'll include this in the refactoring. I'll also add a new environment `ruledtabular` which does nothing except to issue a warning. As ruledtabular always surrounds a tabular environment, the table will still be displayed. 

According to [this answer on stackexchange](https://tex.stackexchange.com/a/20898/126855) the ruledtabular environment would also spread the table over the whole page. In this case I will recommend the use of `table*` in the warning message.

**Comment by cgogolin** [comment]
solved in 4f97f75
